python/mxnet/rnn/rnn_cell.py

```diff
@@ -117,21 +118,29 @@ def reset(self):
         self._counter = -1
 
     def __call__(self, inputs, states):
-        """Construct symbol for one step of RNN.
+        """Unroll the RNN for one time step.
 
         Parameters
         ----------
         inputs : sym.Variable
             input symbol, 2D, batch * num_units
-        states : sym.Variable
-            state from previous step or begin_state().
+        states : list of sym.Variable
+            RNN state from previous step or the output of begin_state().
 
         Returns
         -------
         output : Symbol
-            output symbol
-        states : Symbol
-            state to next step of RNN.
+            output symbol on unrolling this time step.
```

Comment:
Please captalize "Output". Can you be a little clearer? What does "on unrolling this time step". Do you mean "Symbol corresponding to the output from the RNN on this time step." Might that be better?